tests/helpers.py

```diff
@@ -0,0 +1,19 @@
+from asynctest import Mock, CoroutineMock
+
+from asyncworker.connections import AMQPConnection
+from barterdude import BarterDude
+
+
+class MyApp(BarterDude):
+    handlers = (
+        Mock(startup=CoroutineMock(), shutdown=CoroutineMock()),
+    )
+
+
+def get_app():
+    return MyApp(connections=[AMQPConnection(  # nosec
+                name="conn1",
+                hostname="localhost",
+                username="username",
+                password="pass"
```

Comment:
Não deveria ser guest/guest?

Reply:
It's just a "mock"... No connection happens